src/components/NewClusterModal.js

```diff
@@ -41,8 +42,10 @@ const machineConfigsEqual = (a, b) => {
 // distilled from https://github.com/docker/distribution/blob/95daa793b83a21656fe6c13e6d5cf1c3999108c7/reference/regexp.go
 const imageValidationRegexp = /^[A-Za-z0-9]+[\w./-]+(?::\w[\w.-]+)?(?:@[\w+.-]+:[A-Fa-f0-9]{32,})?$/
 
+const validMachineTypes = _.filter(({ memory }) => memory >= 4, machineTypes)
```

Comment:
I wish we could point to some external place where this was officially specified